hkmc2/shared/src/main/scala/hkmc2/bbml/bbML.scala

```diff
@@ -295,9 +295,10 @@ class BBTyper(using elState: Elaborator.State, tl: TL):
       val nestCtx1 = ctx.nest
       val nestCtx2 = ctx.nest
       val patTy = pattern match
-      case Pattern.ClassLike(sym, _, _, _) =>
-        val (clsTy, tv, emptyTy) = sym.asCls.flatMap(_.defn) match
-        case S(cls) =>
+      case pat @ Pattern.ClassLike(_, _, _, _) =>
```

Comment:
Bunches of underscores are an anti-pattern
```suggestion
      case pat: Pattern.ClassLike =>
```